osquery/tables/system/windows/os_version.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+*  Copyright (c) 2014-present, Facebook, Inc.
+*  All rights reserved.
+*
+*  This source code is licensed under the BSD-style license found in the
+*  LICENSE file in the root directory of this source tree. An additional grant
+*  of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#include "osquery/core/conversions.h"
+#include "osquery/core/windows/wmi.h"
+#include <osquery/tables.h>
+
+namespace osquery {
+namespace tables {
+
+#define WMI_QUERY "SELECT CAPTION,VERSION FROM Win32_OperatingSystem"
```

Comment:
Is there any particular reason you have this as a `#define`? With most of the WMI queries we've been using `std::string` defined next to the Request. Do you mind moving this to right before you make your request obj?